src/client/app/helpers/strings.ts

```diff
@@ -115,6 +118,7 @@ export class Strings {
     /** NEEDS STRING REVIEW */
     static readonly sharePublicSnippetConfirm = 'Are you sure you want to share this snippet as a new public gist?';
     static readonly sharePrivateSnippetConfirm = 'Are you sure you want to share this snippet as a new secret gist?';
+    static readonly changeEnvironmentConfirm = 'You are about to change your Script Lab environment and will not have access to your saved snippets until you return to this environment. Are you sure you want to proceed?';
```

Comment:
Good string!  But I would replace "saved snippets" with "saved local snippets", to clarify the difference about local workspaces vs. things saved as gists